Fastpack/FlatPacker.ml

```diff
@@ -854,22 +858,28 @@ let pack ?(cache=Cache.fake ()) (ctx : Context.t) channel =
           total_modules := List.concat [ !total_modules; new_modules; ];
           Lwt.return_unit
   in
-  let mode =
-    Printf.sprintf "var __DEV__ = %s;\n"
-      @@ if ctx.mode = Mode.Development then "true" else "false"
+  let%lwt () = pack ctx MDM.empty in
+
+  let bundle = channel
+    |> Lwt_io.position
+    |> Int64.to_int
+    |> Lwt_bytes.extract bytes 0
+    |> Lwt_bytes.to_string
```

Comment:
What's the perf implication of this?

Reply:
@andreypopp Functions are hoisted, variables too, but `undefined`.

Reply:
@TrySound if it is only `import()` calls - it should be fine. It only executes in `Promise.resolve` hence function `__fastpack_import__` will be evaluated at that time

Reply:
```
$ ./fpack_master src/index.js
Packed in 3.041s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_master src/index.js
Packed in 3.069s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_master src/index.js
Packed in 3.048s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_master src/index.js
Packed in 3.086s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_new src/index.js
Packed in 3.051s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_new src/index.js
Packed in 3.100s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_new src/index.js
Packed in 3.058s. Number of modules: 808. Cache: no. Mode: production

$ ./fpack_new src/index.js
Packed in 3.229s. Number of modules: 808. Cache: no. Mode: production
```